x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alert_create_flyout.ts

```diff
@@ -179,7 +178,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
     it('should show save confirmation before creating alert with no actions', async () => {
       const alertName = generateUniqueKey();
-      await defineAlert(alertName);
+      await defineAlwaysFiringAlert(alertName);
```

Comment:
The test would occasionally receive `TimeoutError: Waiting for element to be located By(css selector, #indexSelectSearchBox)` when defining an index threshold alert before saving.

The same alert is defined in an above test `should create an alert` with no flakiness so I'm not really sure what's causing the flakiness in this test. Switched to defining a simpler alert with less click interaction for now.